validator/specs/lib/validatePntsSpec.js

```diff
@@ -136,6 +136,18 @@ describe('validate pnts', function() {
         expect(validatePnts(pnts)).toBe('Feature table property BATCH_LENGTH must be less than or equal to POINTS_LENGTH.');
     });
 
+    it('returns error message if any BATCH_ID is greater than BATCH_LENGTH: ', function() {
+        var pnts = createPnts({
+            featureTableJson : {
+                POINTS_LENGTH : 1,
+                POSITION : [0, 0, 0],
+                BATCH_ID : [2],
+                BATCH_LENGTH : 1
```

Comment:
For a slightly stronger test consider testing with `POINTS_LENGTH`/`BATCH_LENGTH` being greater than 1. This will guarantee that the loop iteration is working as it should.